ros_ws/src/car_control/src/car_controller.cpp

```diff
@@ -32,11 +33,7 @@ void CarController::publishDriveParameters(double raw_speed, double raw_angle)
         angle_message.data = angle;
         this->angle_publisher.publish(angle_message);
     }
-    else
-    {
-        std::cout << "not running - ";
-    }
-    std::cout << "speed: " << speed << " | angle: " << angle << std::endl;
+    std::cout << "running: " << this->enabled << " | speed: " << speed << " | angle: " << angle << std::endl;
```

Comment:
Could you replace this call with a [ROS_{DEBUG|INFO|WARNING|...}_STREAM](wiki.ros.org/rosconsole) call?
E.g.
```C++
ROS_DEBUG_STREAM("running: " << this->enabled << " | speed: " << speed << " | angle: " << angle << std::endl);
```